### ISSUE TITLE
Can not log in

### ISSUE THREAD

**Issue by Microbion** [bioworkflows/bioworkflows-help#332]
Sign up an account and use github access don't work.

**Comment by bioworkflows** [comment]
Many thanks for bringing this to our attention. We have reproduced and fixed this configuration error. Could you please try again and let us know if the problem has been resolved? Thanks.

**Comment by Microbion** [comment]
No, it returned 500 error when I login with github. And told me email or password is not correct when I login with email.

**Comment by bioworkflows** [comment]
Thank you for your patience. It looks like the problem was caused by a glitch when populating your profile with GitHub info. We have tried to correct issue but cannot test it here, could you re-try? For the login issue for your normal account, could you try to reset your password? If this does not work, please let us know your username so that we can have a deeper look. 

Note that if you already have a normal account, it is recommended that you go to your account settings and connect your GitHub account from there, then try to login with your github account.

**Comment by Microbion** [comment]
Login does work now. There's another trouble.  Numbers of buttons can't responce in my explorers.  Have tested in Edge 120.0.2210.77 and another explorer with Chromium94.0.4506.71 kernel. For example, add workflow button.
![image](https://github.com/bioworkflows/bioworkflows-help/assets/35793183/d79b7aca-ed0b-469e-91ae-e3df25eefe8d)
And page selection
![image](https://github.com/bioworkflows/bioworkflows-help/assets/35793183/9994502b-0940-4884-b10f-274efaa8dcae)

**Comment by bioworkflows** [comment]
I cannot reproduce the problem with the exact version of Edge on a windows 10 pro system. If you cannot find another Edge browser, could you right click -> inspect -> Console (from the top toolbar), and see if there is any error message when you click the buttons? Also, it would be helpful for you to test on Chrome (which is the main browser for development) to rule out potential issue with your account.

**Comment by Microbion** [comment]
Maybe it‘s a network problem.
![1703139986198](https://github.com/bioworkflows/bioworkflows-help/assets/35793183/a5d1f215-5aa7-4b83-9045-037fc64dcf4a)

**Comment by bioworkflows** [comment]
This is indeed very strange because `unpkg.com` is one of the most popular cdn provider and should be more reliable than our website. If you check the source code of the website, look for lines like

```html
<script src="https://unpkg.com/hyperscript.org@0.9.11"></script>
<script src="https://unpkg.com/htmx.org@1.9.6"></script>
```

you should be able to see the content of these two packages.
